config/opal_check_ofi.m4

```diff
@@ -120,6 +121,16 @@ AC_DEFUN([_OPAL_CHECK_OFI],[
                               [],
                               [opal_ofi_happy=no])])
 
+    AS_IF([test $opal_ofi_happy = yes],
+          [AC_CHECK_MEMBER([struct fi_info.nic],
+                           [opal_check_fi_info_pci=1],
+                           [opal_check_fi_info_pci=0],
+                           [[#include "$with_ofi/include/rdma/fabric.h"]])
+
+           AC_DEFINE_UNQUOTED([OPAL_OFI_PCI_DATA_AVAILABLE],
```

Comment:
Minor nit: it doesn't actually matter (because Autoconf does the Right Thing), but technically the `AC_DEFINE_UNQUOTED` should probably outside of the `AS_IF` block (because you want to define the value regardless of the result of the test).